pytest_plugins/markers.py

```diff
@@ -19,6 +19,7 @@ def pytest_configure(config):
         "build_sanity: Fast, basic tests that confirm build is ready for full test suite",
         "rhel_ver_list: Filter rhel_contenthost versions by list",
         "rhel_ver_match: Filter rhel_contenthost versions by regexp",
+        "no_containers: Disable container hosts from being used in favor of VMs",
```

Comment:
instead `no_contianers` cant we be just specific such, `run_on_vm` or `only_vm`?

Reply:
`no_containers` is the extra layer to disable the behavior dynamically and use the VM.

Reply:
My 0.05€:
I'd prefer `no_containers` to prevent potential confusion of what this mark is doing in case we (very unlikely) introduced bare metal hosts.

Reply:
we could, but I personally think no_containers is more clear about what it's doing.